static/styles/subscriptions.css

```diff
@@ -988,6 +988,10 @@
         .stream_change_property_status {
             margin: 9px auto 3px 3px;
         }
+
+        .grey-box {
+            border: none;
```

Comment:
We should probably adjust the HTML to no longer have the `grey-box` class, right?  The goal here is for this page's HTML to be much like a normal settings page.

Reply:
The separators between the settings and the gap between settings is due to this class. If we agree to remove the separators, but still spacing between them would be nice. I will post a screenshot in a couple of minutes.

Reply:
Below are the two screenshots - 
![Screenshot from 2021-08-06 21-56-19](https://user-images.githubusercontent.com/35494118/128542136-7972d33f-c44a-4143-8cdf-704ab8555791.png)
![Screenshot from 2021-08-06 21-52-29](https://user-images.githubusercontent.com/35494118/128542150-344189dd-7f2b-4ed5-a7ae-ba6af3db3901.png)

I think the bottom one does not looks nice but the first one is still decent.

Reply:
I think it's fine to add a spacer class (or perhaps look for whether ), but we shouldn't have a class called "grey-boy" that doesn't generate a grey box!  

That said we do have spacing around headings in "settings", and I wonder if we need to generalize CSS somewhere.